### DISCUSSION TITLE
"An error has occurred"

### DISCUSSION THREAD

**Discussion by gyanisinha** [microsoft/AzUrlShortener#366] [Q&A]
I keep encountering this error. I have used outlook email and followed password guidelines still. Any suggestions please? Thank you
PS: This is a cool project and I am curious to try it out!

![image](https://user-images.githubusercontent.com/87772005/159911917-bca8ac7f-e7b7-4ba6-8841-dd29a6cbe92e.png)

**Comment by fboucher** [comment]
Yes sorry about this. I'm working on the next version V3.  This version will fix all the security issue.
